gammapy/data/observations.py

```diff
@@ -223,6 +235,15 @@ def create(
             location=location,
         )
 
+        meta = ObservationMetaData(
+            deadtime=1 - deadtime_fraction,
```

Comment:
The names are surprising.... spontaneously, with this subtraction we retrieve the livetime_fraction, and not the deadtime